phpBB/language/en/acp/board.php

```diff
@@ -562,8 +562,8 @@
 	'CONTACT_EMAIL_EXPLAIN'			=> 'This address will be used whenever a specific contact point is needed, e.g. spam, error output, etc. It will always be used as the <samp>From</samp> and <samp>Reply-To</samp> address in emails.',
 	'CONTACT_EMAIL_NAME'			=> 'Contact name',
 	'CONTACT_EMAIL_NAME_EXPLAIN'	=> 'This is the contact name that e-mail recipients will see. If you don’t want to have a contact name, leave this field empty.',
-	'EMAIL_FUNCTION_NAME'			=> 'Email function name',
-	'EMAIL_FUNCTION_NAME_EXPLAIN'	=> 'The email function used to send mails through PHP.',
+	'EMAIL_FORCE_SENDER'			=> 'Force from email address',
+	'EMAIL_FORCE_SENDER_EXPLAIN'	=> 'Force from email address to return email. This might be needed for a correct return address when sending mails on some hosts.<br><em><strong>Warning:</strong> Requires the user that the webserver runs as to be added as trusted user to the sendmail configuration.',
```

Comment:
Thanks for updating the label and explanation, it's already more clear. I've tested the functionality and it works perfectly. I have a few comments regarding displaying the option in the ACP:

The closing `</em>` tag is missing after the EXPLAIN.

<img width="631" alt="move_email_force_sender" src="https://user-images.githubusercontent.com/318762/34049983-91b2cbd4-e1b9-11e7-9b5b-84c234e3d783.png">
Can you move this new option below "Return email address".

After that please change the text of these lines:
``` 
'ADMIN_EMAIL'					=> 'From email address',
'ADMIN_EMAIL_EXPLAIN'			=> 'This will be used as the from address on all emails, the technical contact email address. It will always be used as the <samp>Sender</samp> address in emails.',
``` 
``` 
'EMAIL_FORCE_SENDER_EXPLAIN'	=> 'This will set the <samp>Return-Path</samp> to the from email address instead of using the local user and hostname of the server. This setting does not apply when using SMTP.<br><em><strong>Warning:</strong> Requires the user that the webserver runs as to be added as trusted user to the sendmail configuration.</em>',

```

That way the two options will reference to each other and `ADMIN_EMAIL` will no longer state it's "always used" as return-path.